DEVELOPMENT.md

```diff
@@ -11,21 +11,28 @@ RimSort is built in Python using the [PySide2](https://pypi.org/project/PySide2/
 * [PyInstaller](https://pyinstaller.org/en/stable/): packages application for distribution. Install version `5.7.0` if possible. You can do this by running `pip3 install PyInstaller==5.7.0`.
 * [xmltodict](https://pypi.org/project/xmltodict/): parse RimWorld `xml` files. Install version `0.13.0` if possible. You can do this by running `pip3 install xmltodict==0.13.0`.
 * [toposort](https://pypi.org/project/toposort/): sort mods! Install version `1.9` if possible. You can do this by running `pip3 install toposort==1.9`.
+* [sortedcontainers](https://pypi.org/project/sortedcontainers/) 
```

Comment:
SortedContainers is not actually being used in this release. I was testing it out for the error-checking feature but didn't end up needing to use it.